core/source/Number.mint

```diff
@@ -105,4 +105,14 @@ module Number {
       }
     }
   }
+
+  /*
+  Returns if the given number (year) is a leap year or not.
+
+    Number.isLeapYear(2020) == true
+    Number.isLeapYear(2021) == false
+  */
+  fun isLeapYear (year : Number) : Bool {
+    (year % 4 == 0) && (year % 100 != 0 || year % 400 == 0)
+  }
```

Comment:
This method IMO conceptually belongs more to the `Time` module.

Reply:
Something like `isNumberLeapYear`? Because `Time` already has a `isLeapYear` function and Mint doesn't have function overloading, that's why I moved it to the `Number` module.

Reply:
Yup, sounds about right.